### PR TITLE
#261 add convert command

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #261

### Metanorma PR checklist

 - NO Breaking changes (list related PRs)
 - YES Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - NO External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - NO Gem with native library introduced

### Depends on PRs

 - https://github.com/metanorma/mnconvert-ruby/pull/2
 - https://github.com/metanorma/metanorma-iso/pull/557

### CODE REVIEW

**Comment by ronaldtse** [comment]
Tests failing here @CAMOBAP

File: lib/metanorma/cli/command.rb [hunk 1]
```diff
@@ -77,6 +78,36 @@ def collection(filename = nil)
...
+      option :output_format,
+             type: :string,
+             desc: "Output format: xml|adoc|iso|niso"
+      option :output_file, type: :string, desc: "Output file"
+      option :imagesdir,
+             type: :string,
+             desc: "For STS input only: folder with images"
+      option :input_format,
+             type: :string,
+             desc: "Input format: metanorma|sts|rfc"
```
Comment (ronaldtse) [inline lib/metanorma/cli/command.rb:91]: What does "metanorma" mean? Also, there is NISO STS vs ISOSTS that are different formats. Ping @Intelligent2013

Comment (CAMOBAP) [reply lib/metanorma/cli/command.rb:91]: I got that values from `mnconvert.jar --help`

**Comment by Intelligent2013** [comment]
> What does "metanorma" mean? Also, there is NISO STS vs ISOSTS that are different formats.
@ronaldtse "metanorma" means metanorma xml format.
NISO STS is backwards-compatible superset of ISO STS, additional elements and attributes were added into it. So no need to specify concrete version of sts - "niso" or "iso".

**Comment by ronaldtse** [comment]
Thanks @Intelligent2013 . Just going to merge this as the test failures are unrelated.

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**